Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -472,29 +472,51 @@ theorem map {v' : Valuation R Γ₀} (f : Γ₀ →*₀ Γ'₀) (hf : Monotone f
 theorem comap {S : Type*} [Ring S] (f : S →+* R) (h : v₁.IsEquiv v₂) :
     (v₁.comap f).IsEquiv (v₂.comap f) := fun r s => h (f r) (f s)
 
-theorem val_eq (h : v₁.IsEquiv v₂) {r s : R} : v₁ r = v₁ s ↔ v₂ r = v₂ s := by
-  simpa only [le_antisymm_iff] using and_congr (h r s) (h s r)
+variable (h : v₁.IsEquiv v₂) {x y : R}
+include h
 
-theorem ne_zero (h : v₁.IsEquiv v₂) {r : R} : v₁ r ≠ 0 ↔ v₂ r ≠ 0 := by
-  have : v₁ r ≠ v₁ 0 ↔ v₂ r ≠ v₂ 0 := not_congr h.val_eq
-  rwa [v₁.map_zero, v₂.map_zero] at this
+theorem le_iff_le : v₁ x ≤ v₁ y ↔ v₂ x ≤ v₂ y :=
+  h x y
```

Comment:
Author's note: I have opted to include this theorem so that people don't have to "remember the definition"

Reply:
Can you include this note in the docstring, thanks.

Reply:
Done.